lib/app/components/pages/search_filter/controllers/search_filter_controller.dart

```diff
@@ -1,16 +1,48 @@
 import 'package:get/get.dart';
 import 'package:favoritism_communication/app/data/gender/gender.dart';
+import 'package:favoritism_communication/app/repository/genre_repository.dart';
 
 class SearchFilterController extends GetxController {
   final Rx<Map<Gender, bool>> _gender = Rx<Map<Gender, bool>>({});
+  final Rx<List<String>> _genreIds = Rx<List<String>>([]);
+  final Rx<Map<String, Map<String, dynamic>>> _genreMap = Rx<Map<String, Map<String, dynamic>>>({}); // idとjsonデータのマップ
 
   @override
-  void onInit() {
-    super.onInit();
+  void onInit() async{
+    updateGenreMap();
```

Comment:
```suggestion
  Future<void> onInit() async{
    await updateGenreMap();
```
すいません、レビュー遅くなりました。
`Future<void>`にしましょう。
あと`await`キーワードつけ忘れてるみたいなんで、つけておいてください。

Reply:
@king-kazu39 
ありがとうございます！
修正しました！